test/package.test.ts

```diff
@@ -0,0 +1,16 @@
+import { expect } from 'chai';
+
+import StrategyDefaultImport, { Strategy as StrategyNamedImport } from '..';
+
+describe('@superfaceai/passport-twitter-oauth2 package', function () {
+  describe('when using in TypeScript project', () => {
```

Comment:
typo:

```suggestion
  describe('when used in TypeScript project', () => {
```

or just:

```suggestion
  describe('in TypeScript project', () => {
```